cassandra/hooks/run

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+exec 2>&1
+
+#{{#if cfg.runtime.cassandra_java_opts}}
+#   JVM_OPTS -- Additional arguments to the JVM for heap size, etc
+#{{/if ~}}
+export JAVA_HOME="$(hab pkg path core/jre8)"
```

Comment:
There's a `{{pkgPathFor 'core/jre8'}}` helper, so you don't need to depend on `hab` or shell out to it.